config/jobs/kubernetes/cloud-provider-aws/cloud-provider-aws-config.yaml

```diff
@@ -0,0 +1,38 @@
+presubmits:
+  kubernetes/cloud-provider-aws:
+  - name: pull-cloud-provider-aws-check
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181105-66d3bcb98-master
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
please write new jobs using the pod-utils, since @d-nishi is asking to get this in urgently, let's fix that in a follow-up.

we plan to eliminate the scripts these flags go to eventually, for the prow integrated solution instead, this script is a bit of a mess.

docs here: https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#writing-a-prowjob-that-uses-pod-utilities

Reply:
Thanks @BenTheElder! I'll follow up later today or Monday at the latest with this fix.